### DISCUSSION TITLE
SAML single sign-on

### DISCUSSION THREAD

**Discussion by prashant0700** [community/community#61922] [Enterprise]
### Select Topic Area

Question

### Body

Hi

Can anyone please advise if we can implement SSO in GitHub just at Org level if we don't want to implement it at Enterprise level.
We are using Enterprise Cloud.

**Comment by jge162** [comment]
Hello @prashant0700 Yes you can implement SSO at the organization level in GitHub Enterprise Cloud without forcing all members to use it. [documrntstion](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/enabling-and-testing-saml-single-sign-on-for-your-organization)
